tools/scripts/cargo_raze.sh

```diff
@@ -5,3 +5,7 @@ cd "$(git rev-parse --show-toplevel)"
 cd third_party/cargo && cargo raze
 )
 ./tools/scripts/format_build_files.sh
```

Comment:
Is this intermediate call to `format_build_files.sh` needed for the subsequent call to `add_crates_shas.rb`?

Reply:
Resolved via DM: add_crates_shas.rb is sensitive to the formatting of build files, so both calls are needed.